client/lib/cart-values/cart-items.js

```diff
@@ -696,6 +696,16 @@ function isNextDomainFree( cart ) {
 	return !! ( cart && cart.next_domain_is_free );
 }
 
+function isDomainBeingUsedForPlan( cart, domain ) {
+	if ( cart && domain && hasPlan( cart ) ) {
+		const domainProducts = getDomainRegistrations( cart ).concat( getDomainMappings( cart ) ) || [],
```

Comment:
It is truthy yeah, my initial attempt is to ensure that variable is an array so I can call array functions without null-checking but they already ensured. Good catch.